cpu/cc2538/include/cpu_conf.h

```diff
@@ -43,6 +43,32 @@ extern "C" {
 #define CPU_HAS_BITBAND                 (1)
 /** @} */
 
+
+/**
+ * @brief   Flash page configuration
+ * @{
+ */
+#define FLASHPAGE_SIZE          (2048U)
+/* Last page holds the CCA region for safety this will not be allowed
+   to be erased */
+#ifndef FLASHPAGE_CC2538_CCA_PAGE
+#define FLASHPAGE_CC2538_CCA_PAGE   (0)
+#endif
```

Comment:
Does any board use this yet?

Reply:
Nope, I disabled it by default, but left the door open if someone ever needs to do this. I found it risky to do when using OTA for example So I disabled it by default.

Reply:
Ah so this *disables* writing to the last page by default.